### DISCUSSION TITLE
Airflow 2.6.3 Web UI open very slow (it's cost my 1 minutes)

### DISCUSSION THREAD

**Discussion by hrbust-liu** [apache/airflow#40731] [Q&A]
I did some tests to verify
Some people's laptops access very quickly, while some people access very slowly.
Windows is generally very fast, while Macs (Intel/ARM) are generally very slow. Switching browsers doesn't help much.
Using chatgpt to deploy go web services on the server, access is very fast.
Tcpdump investigation found that the server's syn ack packet always fails to come back during the TCP handshake

**Comment by potiuk** [comment]
If it's TCP level - it cannot be Airflow. You should look at whatever way you deploy things on Mac. Can't help with that I am afraid. Maybe if you specify the exact configuration and what you do on Mac to run airlfow, but also generally speaking Mac is only recommended for developing airflow itself, not for running it, because it's not production ready on Mac - so running Airflow on Mac reliably requires using containers and linux VM in general.

**Reply by hrbust-liu** [reply]
You are right! It's centos 3.10 tcp_tw_recycle configuration conflicts with macos tcp_timestamp_random
